docs/source/quickstart/index.rst

```diff
@@ -20,6 +20,66 @@ To install Cloud Custodian, just run::
 
   $ pip install c7n
 
+
+.. _write-policy:
+
+Write your first policy
+-----------------------
+
+A policy specifies the following items:
+
+* The resource to run the policy against
+* The resource state or states the policy checks (filters)
```

Comment:
its a bit more involved then just state, i'd probably say resource attributes, but filters can be fairly arbitrary from cloud watch metrics, to does it have an alarm configured (absence of state).